{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -22,10 +22,23 @@ classifiers = [
 dependencies = []
 requires-python = ">=3.9"
 
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.version]
+path = "{{ cookiecutter.__project_src_path }}/__init__.py"
+
+[tool.hatch.build.targets.wheel]
+packages = ["{{ cookiecutter.__project_src_first }}"]
+
```

Comment:
is this really necessary?

Reply:
defining a build-system is necessary if you want editable installs (which is convenient for development) https://docs.astral.sh/uv/concepts/projects/sync/#editable-installation
the default at the time was hatching when Alexis made his original PR but perhaps we should switch to the uv backend since that's now the default and stable.

Reply:
It is not really stable though yet, it's not 1.x

Reply:
the maintainers claim it's stable since 0.7.19 and ready for production use even though it's not 1.x.x (https://github.com/astral-sh/uv/issues/3957#issuecomment-3033045054) and also made it the default backend.

the only reason not to use it is if you need some fancier features that it does not have implemented (yet) but i think using it for cookie cutter is a good defaul. if you know what you're doing or have some other special use case, you can just modify the project after you make the template